### PR TITLE
Frontend-All users

### PR DESCRIPTION
**Pull request by codegracie**
--my repo got messed up, had to fix it first.
--this is frontend part only

### CODE REVIEW

**Comment by kian1213** [comment]
Okay

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <div id="page-inner">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h2 class="page-header">
+             <i class="fa fa-file-image-o"></i><%= t ".All_users" %>
+            </h2>
+          </div>
+              <table class="table table-striped table-advance table-hover">
```
Comment (MuhammadTamzid) [inline app/views/users/index.html.erb]: indent

File: app/controllers/admin/courses_controller.rb [hunk 2]
```diff
@@ -32,6 +32,7 @@ def course_params
     params.require(:course).permit :user_id, :title, :description, :start_date, :end_date,
       :status, course_subjects_attributes: [:id, :course_id, :subject_id, :_destroy],
       supervisor_courses_attributes: [:id, :user_id, :course_id, :_destroy]
+      user_ids: []
```
Comment (MuhammadTamzid) [inline app/controllers/admin/courses_controller.rb]: Comma missing

File: app/models/course.rb [hunk 3]
```diff
@@ -12,8 +12,11 @@ class Course < ActiveRecord::Base
   validates :status, presence: true
 
   accepts_nested_attributes_for :course_subjects, reject_if:
-    lambda {|a| a[:subject_id].blank?}, allow_destroy: true
+    lambda {|attr| attr[:subject_id].blank?}, allow_destroy: true
```
Comment (MuhammadTamzid) [inline app/models/course.rb]: attr -> attribute and same above

Comment (codegracie) [inline app/models/course.rb]: this part was introduced to me by the trainer..

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  <div id="page-inner">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h2 class="page-header">
+             <i class="fa fa-file-image-o"></i><%= t ".All_users" %>
+            </h2>
+          </div>
+            <table class="table table-striped table-advance table-hover">
```
Comment (dieunb) [inline app/views/users/index.html.erb]: indent

File: app/models/course.rb (hunk 3 above)
Comment (MuhammadTamzid) [inline app/models/course.rb]: ok, using flul name is good

**Pull request merged**